x11-drivers/nvidia-drivers/files/bin/setup-nvidia

```diff
@@ -3,9 +3,8 @@ set -euo pipefail
 
 . /usr/share/flatcar/release
 . /usr/share/flatcar/update.conf
+. /usr/share/nvidia/release
```

Comment:
I'm on the fence here, but I think it would be better if this file was also in /usr/share/flatcar, so that it's clear to the user that it's a flatcar specific file...

Reply:
Makes sense. we can rename it to `/usr/share/flatcar/nvidia-metadata`?

Reply:
Sounds good to me.